modules/c++/mem/include/mem/ScopedCopyablePtr.h

```diff
@@ -106,11 +106,18 @@ class ScopedCopyablePtr
     {
         return !(*this == rhs);
     }
-    
-    explicit operator bool() const 
-    { 
-        return get() == NULL ? false : true; 
+
+    // explicit operators not suppored until C++11
+    bool toBool() const
+    {
+        return get() == NULL ? false : true;
     }
+#ifdef __CODA_CPP11
```

Comment:
You need to add one of the _config headers to get this

Reply:
For non CPP11 this should be valid

#ifdef __CODA_CPP11
    explicit operator bool() const
    {
        return toBool();
    }
#else
    operator bool() const
    {
        return toBool();
    }
#endif

Reply:
OK so in that case @JonathanMeans can you put out a new PR that collapses this to...
<remove `toBool()`>

```
#ifdef _CODA_CPP11
explicit
#endif
operator bool() const
{
    return get() == NULL ? false : true;
}
```